app/assets/stylesheets/custom.scss

```diff
@@ -11,3 +11,14 @@ body {
 .glyphicon.selected {
   color: deepskyblue;
 }
+
+#twit_message {
+  resize: none;
+  width: 215px;
+}
+
+#reply_message {
```

Comment:
Avoid using id selectors<br>Selector `reply_message` should be written in lowercase with hyphens